lib/uplinkc/testdata/object_test.c

```diff
@@ -19,7 +19,15 @@ void handle_project(ProjectRef project) {
     char **err = &_err;
 
     char *bucket_name = "test-bucket";
-    char *enc_ctx = "12VtN2sbbn9PvaEvNbNUBiSKnRcSUNxBADwDWGsPY7UV85e82tT6u";
+
+    uint8_t *salted_key = project_salted_key_from_passphrase(project,
```

Comment:
`salted_key` was `malloc`ed, so we should free it, I think

Reply:
:+1: